cos-nvidia-gpu-installer/installer.sh

```diff
@@ -47,13 +47,12 @@ BIN_OUTPUT_DIR="${BASE_DIR}/bin"
 KERNEL_SRC_DIR="/lakitu-kernel"
 
 NVIDIA_DRIVER_DIR="/nvidia"
-NVIDIA_DRIVER_VERSION="375.26"
+NVIDIA_DRIVER_VERSION="375.66"
 
-# Source: https://developer.nvidia.com/cuda-downloads
-NVIDIA_CUDA_URL="https://developer.nvidia.com/compute/cuda/8.0/Prod2/local_installers/cuda_8.0.61_375.26_linux-run"
-NVIDIA_CUDA_MD5SUM="33e1bd980e91af4e55f3ef835c103f9b"
-NVIDIA_CUDA_PKG_NAME="cuda_8.0.61_375.26_linux.run"
-NVIDIA_DRIVER_PKG_NAME="NVIDIA-Linux-x86_64-375.26.run"
+# Source: https://www.nvidia.com/Download/index.aspx?lang=en-us
+NVIDIA_DRIVER_URL="https://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVER_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
+NVIDIA_DRIVER_MD5SUM="9cf2c0a342a088211f25c5d8137097e6"
+NVIDIA_DRIVER_PKG_NAME="NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
 
 check_nvidia_device() {
     lspci
```

Comment:
Unrelated to this PR, but why do we have this `lspci` command?

Reply:
IIRC, it is to check if lspci is available within the sandbox that's running the script. If the script is only going to deployed within a controlled container, we dont need it.

Reply:
Looks like it is to make sure that we exit with non-zero exit code if `lspci` doesn't exist.

If `lspci` doesn't exist, we exit with non-zero code.
If `lspci` exists but nvidia device doesn't exist, we exit with zero code.
If `lspci` exists and nvidia device also exists then we continue.